tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/commons/js/menu-bar/export-menu.js

```diff
@@ -28,12 +28,12 @@ define(([], function () {
                     id: "export-for-docker",
                     shortcuts: {
                         mac: {
-                            key: "command+shift+e",
-                            label: "\u2318\u21E7E"
+                            key: "option+d",
+                            label: "\u2325D"
                         },
                         other: {
-                            key: "ctrl+shift+e",
-                            label: "Ctrl+Shift+E"
+                            key: "alt+d",
```

Comment:
Please double check whether we have use this combination else where

Reply:
Yes, Alt+D or Alt+K  is not used anywhere